.prettierignore

```diff
@@ -0,0 +1,2 @@
+*.yml
+*.yaml
```

Comment:
`circleci/config.yaml` uses double quotes
But prettier is configured to use single quotes, see `.prettierrc`.

We either need to change all yaml files to single quotes or simply ignore all yaml files.

I use `"editor.formatOnSave": false` in my IDE, which triggers this behaviour.